userspace/libsinsp/test/public_sinsp_API/interesting_syscalls.cpp

```diff
@@ -362,27 +371,33 @@ TEST(interesting_syscalls, get_all_ppm_sc)
 
 /* This test asserts that `get_syscalls_names` correctly retrieves all the syscalls names
  */
-TEST(interesting_syscalls, get_syscalls_names)
+TEST(interesting_syscalls, get_sc_names)
 {
-	std::set<std::string> orderd_syscall_names_matching_set;
+	std::set<std::string> orderd_sc_names_matching_set;
 	libsinsp::events::set<ppm_sc_code> ppm_sc_set;
 
 	/* Here we don't need ifdefs, our ppm_sc codes are always defined. */
 	ppm_sc_set.insert(PPM_SC_KILL);
-	orderd_syscall_names_matching_set.insert("kill");
+	orderd_sc_names_matching_set.insert("kill");
 
 	ppm_sc_set.insert(PPM_SC_READ);
-	orderd_syscall_names_matching_set.insert("read");
+	orderd_sc_names_matching_set.insert("read");
+
+	ppm_sc_set.insert(PPM_SC_SYS_ENTER);
+	orderd_sc_names_matching_set.insert("sys_enter");
+
+	ppm_sc_set.insert(PPM_SC_SCHED_PROCESS_FORK);
+	orderd_sc_names_matching_set.insert("sched_process_fork");
```

Comment:
Just to really have more coverage even if we think it's not necessarily needed would recommend also testing removing a TP SC ... if you have more ideas, I am always happy to rather see more / redundant tests than too few.